api/src/main/java/org/hawkular/accounts/api/internal/CassandraSessionInitializer.java

```diff
@@ -67,12 +68,19 @@ public void init() {
         sessionFuture = executor.submit(cassandraSessionCallable);
     }
 
+    @PreDestroy
+    public void destroy() throws Exception {
+        logger.shuttingDownCassandraDriver();
+        sessionFuture.get().getCluster().closeAsync();
```

Comment:
As per their javadoc, calling "close" on an already closed cluster/session is a noop.